packages/guideline-blocks-settings/src/components/BlockItemWrapper/BlockItemWrapper.tsx

```diff
@@ -0,0 +1,81 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { PropsWithChildren, useEffect, useRef, useState } from 'react';
+import { joinClassNames } from '../../utilities';
+import Toolbar from './Toolbar';
+import { BlockItemWrapperProps, ToolbarItem } from './types';
+
+export const BlockItemWrapper = ({
```

Comment:
It is a wrapper we have been reusing a lot. Adds a purple outline and toolbar to an element. Do you think it might make sense to keep it in our own shared?

Reply:
Isn't the outline supposed to be added by the Content Manager? 🤔

Reply:
It is an outline within the block. E.g. do&dont items inside the do&dont block.